hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson

```diff
@@ -327,7 +327,47 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CSR.SHADOW",
```

Comment:
I see this PR adds `CSR` but we already have `CONFIG` for the same purpose (software supplied config through comportable reg_top). Should we stick to one?

Reply:
Yeah let's use that one instead of adding another asset label.